sympy/solvers/ode/tests/test_systems.py

```diff
@@ -781,6 +893,53 @@ def test_linear_3eq_order1_type4_slow():
     dsolve(eq1)
 
 
+@slow
+@XFAIL
+def test_linear_neq_order1_type2_fail():
+    i, r1, c1, r2, c2, t = symbols('i, r1, c1, r2, c2, t')
+    x1 = Function('x1')
+    x2 = Function('x2')
+    eq1 = r1*c1*Derivative(x1(t), t) + x1(t) - x2(t) - r1*i
+    eq2 = r2*c1*Derivative(x1(t), t) + r2*c2*Derivative(x2(t), t) + x2(t) - r2*i
+    eq = [eq1, eq2]
+    sol = dsolve(eq)
+    assert checksysodesol(eq, sol) == (True, [0, 0])
+
+
+@slow
+@XFAIL
+def test_linear_new_order1_type2_fail_too_slow():
+    if ON_TRAVIS:
```

Comment:
Multiple XFAIL tests should not be grouped together in a single test because then we won't know when it passes.